### PR TITLE
Fixes an issue with the reportinglib CI (#472)

### PR DESCRIPTION
**Pull request by sergiorg-hpc**
* Update reporting_test.sh.in
* Bump coreneuron datasets' version from 1.2 to 1.4

CI_BRANCHES:NEURON_BRANCH=master,

### CODE REVIEW

**Comment by pramodk** [comment]
@alexsavulescu : we haven't rebased / cherry-picked the fix here yet. I will do that next, after #473.

File: tests/integration/reportinglib/reporting_test.sh.in [hunk 1]
```diff
@@ -1,4 +1,7 @@
 #! /bin/sh
+
+set -e -o pipefail
+
```
Comment (pramodk) [inline tests/integration/reportinglib/reporting_test.sh.in:4]: so there were no _3 dat files meaning report tests were not running at all? 🤔

**Review by pramodk — APPROVED** [review]

**Comment by alexsavulescu** [comment]
There is a bug with the displayed jobs(because all are successful but also displayed as required?)

**Comment by pramodk** [comment]
But if you see the names, they are bit different? not sure if that is actually mixed up somehow...!

**Comment by alexsavulescu** [comment]
> But if you see the names, they are bit different? not sure if that is actually mixed up somehow...!

Looks like they are not part of the workflow (`CoreNEURON CI`). 
We should open a new PR from a different (copied) branch and see if that fixes it

**Comment by sergiorg-hpc** [comment]
Closing this PR, as it is being replace by #483

**Pull request closed without merging**